stylelint.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
 		"stylelint-order"
 	],
 	"rules": {
+		"indentation": 4,
```

Comment:
Was wondering why this was changed to 4?

Reply:
The rule of using 4 spaces for indentations is applied in other IDRC projects. I wonder it would be useful to be in sync so developers don't have to change their IDE settings when moving across projects.

Other rules for js files such as indentation using 4 spaces for indentations and having semicolon at the end of lines are also applied in other projects. I lean towards applying the same to this project but let's discuss at today's check-in. Thanks.

Reply:
Okay sounds good. The rest of this PR lgtm

Reply:
@cindyli Do you use [editorconfig](https://editorconfig.org) in your IDE? This might be a good solution to allow the editor to adopt whatever the code styling conventions are for a given project. I don't have any issue with this change, though.